core/src/main/java/org/apache/accumulo/core/clientImpl/ThriftTransportKey.java

```diff
@@ -91,7 +91,7 @@ public boolean equals(Object o) {
   }
 
   public final void precomputeHashCode() {
-    hashCode();
+    var unusedRetVal = hashCode();
```

Comment:
Instead of doing this, could just do this computation in the constructor and delete this method. The only place where this is used is immediately after construction in one case, and the only other time this object is constructed is in ServerClient where it is not going to hurt to precompute.

Reply:
Removed precomputeHashCode method and moved call to hashCode into constructors.

Reply:
Assigning the result of the hashCode to final int removes EP error. Is that going to cause other issues in Eclipse or elsewhere? If so, can re-work to make use of log.trace message or find another means to use return value.  Did not want to add assertNonNull or similar in non-test code.